src/map/handler/Map.Drag.js

```diff
@@ -96,12 +96,12 @@ export var Drag = Handler.extend({
 	},
 
 	_onDown: function () {
```

Comment:
If the body of `_onDown` is empty, we might as well remove the method definition and the `this._draggable.on('down': this._onDown)` line up.